apps/fluent-tester/package.json

```diff
@@ -117,7 +117,7 @@
     "webdriverio": "7.23.0"
   },
   "peerDependencies": {
-    "@office-iss/react-native-win32": ">=0.68.0 <0.68.6"
+    "@office-iss/react-native-win32": ">=0.68.0 <=0.68.8"
```

Comment:
Should we remove the range now that we can move past 0.68.6? That was to keep us from getting an update with a bug in it right?

Reply:
Oh sure, I wasn't sure so I just kept the format.